ClientApp/viking-vault/src/App.css

```diff
@@ -1,5 +1,11 @@
 .App {
   text-align: center;
+  width:'100%';
```

Comment:
No need to put the width property value inside of quotes ``'100%'`` --> ``100%``